pom.xml

```diff
@@ -1001,11 +1004,26 @@
         <artifactId>maven-failsafe-plugin</artifactId>
         <executions>
           <execution>
-            <id>run-integration-tests</id>
+            <id>run-minicluster-integration-tests</id>
             <goals>
               <goal>integration-test</goal>
               <goal>verify</goal>
             </goals>
+            <configuration>
+              <skip>${accumulo.skipMiniClusterInFailsafe}</skip>
+              <groups>${accumulo.miniclusterTests}</groups>
+            </configuration>
+          </execution>
+          <execution>
```

Comment:
I'm confused why there are two executions of the plugin. Won't the logs from these two executions conflict? How does failsafe match up the two different integration-test mojo outputs with their corresponding verify mojo work if both execute?